terraform/alerting/dashboards/e2e.json

```diff
@@ -0,0 +1,52 @@
+{
+  "displayName": "e2e",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [
+      {
+        "title": "OpenCensus/en-verification-server/api/issue/codes_issued_count (filtered) (gr...",
```

Comment:
...?

Reply:
This JSON was exported from existing dashboards. Updated to have a better name.